sgt-back-end/index.js

```diff
@@ -0,0 +1,167 @@
+const express = require('express');
+const pg = require('pg');
+
+const app = express();
+
+const db = new pg.Pool({
+  connectionString: 'postgres://dev:dev@localhost/studentGradeTable',
+  ssl: {
+    rejectUnauthorized: false
+  }
+});
+
+const expressJson = express.json();
+app.use(expressJson);
+
+app.listen(3000, () => {
+  process.stdout.write('Server is watching on port 3000!');
+});
+
+app.get('/api/grades/', (req, res, next) => {
+  const sql = `
+    select *
+      from "grades"
+  `;
+  db
+    .query(sql)
+    .then(result => {
+      const grades = result.rows;
+      res.json(grades);
+    })
+    .catch(err => {
+      console.error(err);
+      res.status(500).json({
+        error: 'An unexpected error occurred.'
+      });
+    });
+});
+
+app.post('/api/grades/', (req, res, next) => {
+  const newGrades = req.body;
+  if (!newGrades.name) {
```

Comment:
This variable name is a little misleading. We're only dealing with one record so `newGrade` (singular) would be better.